src/package.json

```diff
@@ -12,6 +12,7 @@
     "coverage": "vitest run --coverage",
     "cypress:open": "cypress open",
     "cypress:run_headless": "cypress run",
+    "cypress:test": "cypress run --spec cypress/e2e/search.cy.ts",
```

Comment:
Poista tämä. Tätä voi ajaa itse näin `npx cypress run --spec cypress/e2e/search.cy.ts`

Reply:
en